src/Microsoft.IdentityModel.JsonWebTokens/JsonWebTokenHandler.cs

```diff
@@ -1070,13 +1070,19 @@ public virtual TokenValidationResult ValidateToken(string token, TokenValidation
                     var innerToken = ValidateSignature(decryptedJwt, validationParameters);
                     jwtToken.InnerToken = innerToken;
                     var innerTokenValidationResult = ValidateTokenPayload(innerToken, validationParameters);
-                    return new TokenValidationResult
+                    var tokenValidationResult = new TokenValidationResult
                     {
                         SecurityToken = jwtToken,
                         ClaimsIdentity = innerTokenValidationResult.ClaimsIdentity,
                         IsValid = true,
                         TokenType = innerTokenValidationResult.TokenType
                     };
+
+                    var claims = JwtTokenUtilities.CreateDictionaryFromClaims(innerTokenValidationResult.ClaimsIdentity.Claims);
```

Comment:
You should be getting a suggestion from the editorconfig to use the fully qualified type here